README.md

```diff
@@ -40,12 +40,10 @@ $ go run main.go client --address localhost:11111
 
 ### Usage
 
-Use the `--help` flag for usage information.
-
```

Comment:
Would it make sense to also remove the `### Usage` header here? Otherwise you'll end up with an empty section in th doc.